lib/howl/buffer.moon

```diff
@@ -267,6 +268,18 @@ class Buffer extends PropertyObject
       b_end_pos = @byte_offset end_pos
       @_buffer\ensure_styled_to pos: b_end_pos
 
+  mode_at: (pos) =>
+    b_pos = @byte_offset pos
+    marker = @_buffer.styling\get_nearest_style_marker b_pos
+    if marker then mode.by_name marker.name else @mode
+
+  config_at: (pos) =>
+    new_config = config.local_proxy!
```

Comment:
Minor nit but you don't need to create new_config until after the conditional return.